test/ex_machina_test.exs

```diff
@@ -27,7 +27,9 @@ defmodule ExMachinaTest do
   defmodule MyApp.NoSaveFunction do
     use ExMachina
 
-    def factory(:foo), do: %{foo: :bar}
+    factory(:foo) do
```

Comment:
I think I prefer the one-liner, but if you don't like it then that's ok. We can keep it